README.md

```diff
@@ -127,10 +127,10 @@ simulate an install
 -------------------
 ### skip-deploy
 * **BitOps Property:** `skip-deploy`
-* **Environment Variable:** `SKIP_DEPLOY`
-* **default:** `""`
-
-will skip helm execution
+* **Environment Variable:** `HELM_SKIP_DEPLOY`
+* **default:** `none`
+* **Required:** `false`
```

Comment:
I see only one config setting `namespace` in this README mentions the required.

Do we really need to list the `required` property? What's the convention we'd like to go with?

Reply:
All other documentation has the "required" flag. Defaults to false. 

How did you want to handle this?